src/event/event.action.js

```diff
@@ -0,0 +1,28 @@
+import has from 'lodash/has';
```

Comment:
This is the file handling all events action.
I find it handy to have all functions here, instead of cluttering api/api.client.js.

`_fetchEvents` could be inlined in `loadEvents` too, but if you notice later, a lot of function looks just like `loadEvents`, maybe we could make a unique function out of it.